src/test/resources/TestNG.xml

```diff
@@ -7,6 +7,7 @@
         <classes>
             <class name="LoginTests"/>
             <class name="BaseTest"/>
+            <class name="Homework19"/>
```

Comment:
Add baseURL as you have passed baseURL parameters in BeforeMethod in BaseTest class.
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >

<suite name="TestNG">
    <parameter name="baseURL" value="https://qa.koel.app/"></parameter>

    <test name="Example Test" preserve-order="false">

        <classes>
            <class name="LoginTests"/>
            <class name="BaseTest"/>
            <class name="Homework19"/>
        </classes>
    </test>
</suite>
```